### ISSUE TITLE
Display the current playlist in the main page

### ISSUE THREAD

**Issue by MightyCreak** [Sonerezh/sonerezh#153]
Hi!

I think displaying all the songs in the main page isn't very useful. Especially when you start to have a lot of them.

I suggest to display here the current playlist: what are the last 10, 20 songs that you listened to, the current one and the next one that are planned to be played.

Cheers!
And thank you for this excellent piece of software!

**Comment by lGuillaume124** [comment]
Indeed, it could be an interesting approach to improve the main page.
